powerline/segments/common/env.py

```diff
@@ -19,20 +19,23 @@ def environment(pl, segment_info, variable=None):
 
 
 @requires_segment_info
-def virtualenv(pl, segment_info, ignore_venv=False, ignore_conda=False):
+def virtualenv(pl, segment_info, ignore_venv=False, ignore_conda=False, ignored_names=("venv", ".venv")):
```

Comment:
As a very minor comment, can you use a list instead of a tuple for the default argument? (This makes it similar to the rest of the code and is also less confusing when configuring the segment via json, where the param would become a list anyway.)

Other than that, nice work!

Reply:
PyCharm complains about that :disappointed:
Background: When the default argument is a list, it is mutable. This means that _if_ I would change that list within the function (e.g. append a value), this change is permanent. In this case when appending a value within the function this list would grow and grow.

Given that the code is rather small and the values are only read, this is not a problem _here_, but when requiring immutable, I think having an immutable object by default makes this whole thing more explicit

Reply:
Oh indeed, this is a very good point. Let's keep the tuple then.

Reply:
In fact, I just looked again at the other parameters where a list is expected and they all seem to either lack a default argument or have it set to `None`, so having a tuple here is actually fine, I'd say. In any case, the documentation should say `param list ...` though.